src/plugin/context/systemparams/rapier_context_access.rs

```diff
@@ -0,0 +1,125 @@
+use bevy::ecs::system::SystemParam;
+use bevy::prelude::*;
+use std::ops::{Deref, DerefMut};
+
+use super::super::{DefaultRapierContext, RapierContext, RapierContextEntityLink};
+/// Utility [`SystemParam`] to easily access the default world [`RapierContext`] immutably
```

Comment:
It should be made clear that usage of this struct's methods/dereferencing it will cause a panic if the number of worlds that has this component is not 1.